src/state/index.ts

```diff
@@ -34,6 +34,10 @@ export const useIsCapturing = () => useAtom(isCapturing);
 const camera = atom<Camera | undefined>(undefined);
 export const useCamera = () => useAtom(camera);
 
+export type SceneMode = "3d" | "2d" | "columbus";
+const sceneMode = atom<SceneMode | undefined>("3d");
```

Comment:
Is this actually ever undefined? It starts as "3d" and I'm guessing there isn't (or shouldn't be) any way to set it to undefined, right? Or is this based on Cesium types?

Reply:
emmm yes there shouldn't be any way to set it to undefined. i'll update this part and the setter.